src/astro_databricks/plugins/plugin.py

```diff
@@ -141,18 +163,81 @@ def _get_api_client():
     )
 
 
-def _get_launch_task_key(current_task_key: TaskInstanceKey, group_id: str):
-    if group_id:
+def get_task_group_legacy(operator: BaseOperator) -> TaskGroup:
```

Comment:
We could probably benefit by having unit tests for this

Reply:
We agreed to address this as part of a separate ticket: https://github.com/astronomer/astro-providers-databricks/issues/14